lib/getstats.js

```diff
@@ -169,6 +170,7 @@ function standardizeChromeActiveIceCandidatePairStats(stats) {
  * @returns {?StandardizedActiveIceCandidatePairStatsReport}
  */
 function standardizeFirefoxActiveIceCandidatePairStats(stats) {
+
```

Comment:
@syerrapragada Remove extra line here and rename function to `standardizeEdgeAndFirefoxActiveIceCandidatePairStats`?